src/ext/java/org/opentripplanner/ext/digitransitemissions/EmissionsServiceModule.java

```diff
@@ -0,0 +1,19 @@
+package org.opentripplanner.ext.digitransitemissions;
+
+import dagger.Module;
+import dagger.Provides;
+import jakarta.inject.Singleton;
+
+/**
+ * The service is used during application serve phase, not loading, so we need to provide
```

Comment:
Do we have some standard naming for build phase and "runtime"?

Reply:
Good question.